nixos/modules/services/desktops/gnome3/gnome-settings-daemon.nix

```
@@ -0,0 +1,45 @@
+# GNOME Settings Daemon
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.gnome3.gnome-settings-daemon;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.gnome3.gnome-settings-daemon = {
+
+      enable = mkEnableOption "GNOME Settings Daemon.";
+
+      # They're many forks of gnome-settings-daemon
```

I apologize for the inane grammar nitpicking but:

```suggestion
      # There are many forks of gnome-settings-daemon
```

---

No problem, this reads better :+1: